crates/client/rpc-core/src/types/deprecated.rs

```diff
@@ -43,3 +43,17 @@ impl From<EntryPointWrapper> for DeprecatedCairoEntryPoint {
         Self { selector, offset }
     }
 }
+
+impl From<DeprecatedCairoEntryPoint> for EntryPointWrapper {
+    fn from(value: DeprecatedCairoEntryPoint) -> Self {
+        let selector = <[u8; 32]>::from_hex(format_hex(&value.selector)).unwrap();
+        let offset = u128::from_str_radix(&format_hex(&value.offset), 16).unwrap();
+        Self { selector, offset }
+    }
+}
+
+/// Removes the "0x" prefix from a given hexadecimal string and pads it with 0s
+#[inline(always)]
```

Comment:
don't you think this function should live elsewhere ? in a utils primitives or something like that